tutorials/kb_guided_llm_ner/spacy.cfg

```diff
@@ -0,0 +1,19 @@
+[nlp]
+lang = "en"
+pipeline = ["llm","dbpedia-spotlight"]
+
+[components]
+
+[components.llm]
+factory = "llm"
+
+[components.llm.task]
+@llm_tasks = "spacy.NER.v2"
+labels = ["Person", "Organisation"]
+
+[components.llm.model]
+@llm_models = "spacy.GPT-4.v1"
+config = {"temperature": 0.3}
```

Comment:
Why choose 0.3 and not 0.0? Is it on purpose to obtain more possibly wrong cases? (if not I'd recommend changing to 0.0)

Reply:
Not much of purpose there (more copy-paste from another config) so yeah, set to 0.0 now :)